docker/e2e/chaos.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash 
+RED='\033[0;31m'
+NC='\033[0m'
+CYAN='\033[0;36m'
+PURPLE='\033[0;35m'
+GREEN='\033[0;32m'
+
+c=0
+no_sleep () {
+    echo -e "${GREEN}RUN TENDERMINT NO SLEEP CHAOS TEST ${GREEN}"
```

Comment:
I saw the other day that the `-e` argument gets printed out. Are you sure `-e` is valid? It's also not in every echo.

Reply:
originally was using /n new lines to print and had the -e as backslash interpreter, forgot to remove them